src/atoms/constants/accessibility-labels.ts

```diff
@@ -0,0 +1,3 @@
+export enum AccessibilityLabels {
```

Comment:
Can this be moved to an `enums` folder instead of `constants`? Really nitpicky, but it's a little misleading otherwise.

Additionally, I'd move the folder up a level (not nested under `atoms`) - seeing as we're already using it in a molecule, it doesn't seem specific to atom components.

Reply:
Agree. Done. 👍